examples/nemo/scripts/omp_gpu_trans.py

```diff
@@ -76,6 +75,7 @@
 # List of all files that psyclone will skip processing
 FILES_TO_SKIP = [
     "icefrm.f90",  # Has an unsupported implicit symbol declaration
+    "icerst.f90",
```

Comment:
Can you give a reason for this in a comment?

Reply:
The CI succeeded without it, so I removed it.